R/anova_tidiers.R

```diff
@@ -103,22 +103,28 @@ tidy.aov <- function(x, ...) {
 
 
 #' @rdname anova_tidiers
+#' 
+#' @importFrom plyr ldply
+#' 
+#' @import dplyr
+#' 
 #' @export
 tidy.aovlist <- function(x, ...) {
   # must filter out Intercept stratum since it has no dimensions
   if (names(x)[1L] == "(Intercept)") {
     x <- x[-1L]
   }
 
-  # ret <- plyr::ldply(x, tidy, .id = "stratum")
-  ret <- lapply(x, function(a) tidy(stats::anova(a)))
-  ret <- lapply(
-    names(ret),
-    function(a) dplyr::mutate(ret[[a]], stratum = a)
-  )
-  ret <- do.call("rbind", ret)
+  ret <- plyr::ldply(x, tidy, .id = "stratum")
```

Comment:
We're looking to move away from `plyr` at the moment. Would you be willing to use `purrr` here instead?

Reply:
Works for me. I'll switch it up.